src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -249,7 +249,7 @@ object TelemetryQueryStatusFields {
   // Query statement
   val QUERY: String = "query"
   // query ID if available
-  val QUERY_ID: String = "queryid"
+  val QUERYID: String = "queryid"
```

Comment:
query_id, to match max_retry and use_proxy, etc.

Reply:
Will update